ide/docker/Dockerfile

```diff
@@ -5,9 +5,10 @@ ENV DEBIAN_FRONTEND=noninteractive
 # Install build dependencies
 RUN apt-get update && apt-get install -y python3 vim ccache python3-pkgconfig gdb && apt-get clean
 COPY ./provisioning/install-debian-packages.sh /root/install-debian-packages.sh
-RUN /root/install-debian-packages.sh && apt-get clean
 COPY ./provisioning/install-android-tools.sh /root/install-android-tools.sh
-RUN chmod 0755 /root/install-android-tools.sh /root/install-debian-packages.sh
+RUN chmod 0755 /root/install-debian-packages.sh /root/install-android-tools.sh 
```

Comment:
How does changing the order of file names solve anything?

Reply:
You just need to chmod before executing otherwise you got an access deny